src/classes/ADDR_AdminAcc_TEST.cls

```diff
@@ -0,0 +1,891 @@
+        for (Integer i = 0; i < accsAddrs.accs.size(); i++) {
+            accsAddrs.accs[i].BillingStreet = addrs[i].MailingStreet__c;
+            accsAddrs.accs[i].BillingCity = addrs[i].MailingCity__c;
+            accsAddrs.accs[i].BillingState = addrs[i].MailingState__c;
+            accsAddrs.accs[i].BillingPostalCode = addrs[i].MailingPostalCode__c;
+            accsAddrs.accs[i].BillingCountry = addrs[i].MailingCountry__c;
+        }        
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;
+        Test.startTest();
+        update accsAddrs.accs;
+        Test.stopTest();
+        
+        map<Id, Account> mapAccIdAcc = new map<Id, Account>([select Id, Name, BillingStreet, BillingCity, BillingState, 
+        BillingPostalCode, BillingCountry, BillingLatitude, BillingLongitude, is_Address_Override__c, Current_Address__c from Account]);
+        
+        for (Account acc : mapAccIdAcc.values()) {
+            system.assert(acc.BillingStreet.contains('New Street'));
+            system.assert(acc.BillingCity.contains('New City'));
+            system.assertEquals(false, acc.is_Address_Override__c);
+            system.assertNotEquals(null, acc.Current_Address__c);
+        }
+        
+        // verify the new addresses still are Default and we don't have too many
+        list<Address__c> listAddr = [select Id, Default_Address__c, MailingStreet__c, Parent_Account__c, Latest_Start_Date__c, 
+        Latest_End_Date__c from Address__c];
+        system.assertEquals(2 * 2, listAddr.size());
+        for (Address__c addr : listAddr) {
+            boolean isNew = (addr.MailingStreet__c.contains('New Street'));
+            system.assertEquals(isNew, addr.Default_Address__c);
+        }
+    }
+
+    /*********************************************************************************************************
+    @description Insert account w/ multiline street address
+    verify: new default Address created and the street address is broken up into street and street2
+    **********************************************************************************************************/            
+    @isTest
+    public static void insertMultilineStreetAddress() {
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c(Accounts_Addresses_Enabled__c = 
+                                                                    UTIL_Describe.getAdminAccRecTypeID() + ';'));
+    
+        List<Account> accs = UTIL_UnitTestData_TEST.CreateMultipleTestAccounts(2, UTIL_Describe.getAdminAccRecTypeID());
+        for (Account acc : accs) {
+            acc.BillingStreet = 'new street\r\nsecond line';
+            acc.BillingCity = 'new city';
+        }
+        Test.startTest();
+        insert accs; 
+        Test.stopTest();       
+        
+        // verify the Address objects split street.
+        list<Address__c> listAddr = [select Id, MailingStreet__c, MailingStreet2__c, Formula_MailingStreetAddress__c from Address__c];
```

Comment:
If the table in this query contains more than 50'000 records a runtime exception will be thrown. Consider either introducing a `WHERE` clause or contain the number of records redurned, or adding a `LIMIT` clause to cap the maximum number of records returned.